lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/injection/FindMealModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.kinandcarta.lib.find.meal.injection
+
+import android.content.Context
+import com.google.android.gms.location.FusedLocationProviderClient
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.FragmentComponent
+import dagger.hilt.android.qualifiers.ActivityContext
+
+@Module
+@InstallIn(FragmentComponent::class)
+class FindMealModule {
+
+    companion object {
+        @Provides
+        fun provideFusedLocationClient(@ActivityContext context: Context) =
```

Comment:
Here's where some of the magic happens. Hilt uses this @Provides function to create an instance of FusedLocationProvider whenever somewhere that injects it needs it (like in the view model constructor). By adding @ActivityContext into this function, it injects the activity context (not app context) into the function for me so I can use it to create the provider. You can also use @ApplicationContext here

Reply:
Niiice, that will set us up for reuse in Maps Fragment too